spec/integration/cli/db/apply_spec.rb

```diff
@@ -24,6 +24,31 @@
       end
     end
 
+    it "migrates, dumps structure, deletes migrations", if: RUBY_VERSION >= '2.4' do
+      with_project do
+        versions = generate_migrations
+
+        hanami "db apply"
+
+        hanami "db version"
+        expect(out).to include(versions.last.to_s)
+
+        db         = Pathname.new('db')
+        schema     = db.join('schema.sql').to_s
+        migrations = db.join('migrations')
+
+        expect(schema).to have_file_content <<-SQL
+CREATE TABLE `schema_migrations` (`filename` varchar(255) NOT NULL PRIMARY KEY);
+CREATE TABLE `users` (`id` integer NOT NULL PRIMARY KEY AUTOINCREMENT, `name` varchar(255), `age` integer);
+CREATE TABLE sqlite_sequence(name,seq);
```

Comment:
Why do we need this?

Reply:
Maybe if we need this line it is better to include in the other spec instead of duplicating it. What do you think?

Reply:
At first, I checked inclusion of first and second line, then last 2 lines, and found out that earlier ruby with it's sqlite3 wraps table names with `"`. 

Actually, this spec is not about checking exactly this line, but about the execution of `hanami db apply`. So at the end of the day I decided to check a full `got` schema with full `expected` one in extra spec. 

Unfortunately, it leads to almost full duplication of spec implementation. Though it is a valid spec to have, from my point of view, because rspec will ignore another one in runtime and spec can be easily removed in future, when ruby 2.3.x support stopped.

@AlfonsoUceda WDYT?